doc/developer-guide/wasmType_usage.md

```diff
@@ -42,7 +119,7 @@ const a = 100 as i32;
 ```
 
 
-### For array type
+### Used as array element type
```

Comment:
Should also update title in L138?

Reply:
Yeah, my mistake. I will update the titles too.